sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -151,7 +151,7 @@ def get_all_distributions_by_type(dist, metric_id):
     list of :class:`DistributionMetric` objects
   """
   submit_timestamp = time.time()
-  dist_types = ['count', 'max', 'min', 'sum']
+  dist_types = ['count', 'max', 'min', 'sum', 'mean']
```

Comment:
@damccorm Added mean here

Reply:
Looking back through history, it looks like this was intentionally omitted in favor of sum/count - https://github.com/apache/beam/commit/bef40cc02231945bf345065e0fb0703d42fce86c - are we able to just use sum/count instead of mean?

Reply:
Yep. I went through that PR. I use the `LoadTest`  which has helper functions coded already to publish the metrics to the InfluxDB. 

I added it because `DistributionResult` has mean() method and to avoid redundant calculation mean with sum/count

https://github.com/apache/beam/blob/3a4d57eb8976c5f503b32d478a80b1800490f66f/sdks/python/apache_beam/metrics/cells.py#L270

Reply:
I'm good with that